datafusion/core/tests/sqllogictests/test_files/explain.slt

```diff
@@ -70,7 +70,7 @@ CREATE EXTERNAL TABLE aggregate_test_100_with_order (
 STORED AS CSV
 WITH HEADER ROW
 WITH ORDER (c1 ASC)
-LOCATION '../../testing/data/csv/aggregate_test_100.csv';
+LOCATION 'tests/data/aggregate_test_100_with_order.csv';
```

Comment:
I suggest using a name that gives some specific information about what order is present. Something like this, perhaps

```suggestion
LOCATION 'tests/data/aggregate_test_100_order_by_c1_asc.csv';
```